api/desecapi/serializers.py

```diff
@@ -494,12 +494,6 @@ def is_empty(data_item):
         # so for parallel requests, we can get integrity errors due to duplicate keys.
         # This will be considered a 429-error, even though re-sending the request will be successful.
```

Comment:
adjust comment according to change below

Reply:
done